src/App/App.js

```diff
@@ -30,45 +33,46 @@ function App() {
     fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}`)
     .then((res) => {
       if (!res.ok) {
-        console.log("error");
+        displayError(400);
       } else {
         return res.json();
       } 
     })
     .then((data) => setSelectedMovie(data.movie))
+    .catch(err => displayError(500))
   }
 
-
   function showDesc(evt) {
     getSingleMovie(evt.target.id);
     setIsMovieSelected(true);
   }
+
+  function displayHomePage() {
+    setIsMovieSelected(false)
+    setSelectedMovie({})
+  }
+
+  function displayError(responseLevel) {
+    setIsError(true)
+    setResponseLevel(responseLevel)
+  }
   
   return (
     <React.Fragment>
-      <Nav />
-      <React.Fragment>
-        {!isMovieSelected ? (
-          <Main showDesc={showDesc} movieData={movies} />
-        ) : (
-          <Desc movie={selectedMovie} />
-        )}
-      </React.Fragment>
+      {isError ? (<p className="error">ERROR: {responseLevel} </p>) : (
+        <React.Fragment>
+          <Nav />
+          <React.Fragment>
+          {!isMovieSelected ? (
+            <Main showDesc={showDesc} movieData={movies} />
+            ) : (
+            <Desc movie={selectedMovie} displayHomePage={displayHomePage}/>
+          )}
+          </React.Fragment>
```

Comment:
`React.Fragment` seems to be deprecated ([link here](https://legacy.reactjs.org/docs/fragments.html)) and it's now recommended to use [this syntax](https://react.dev/reference/react/Fragment). Also, typically you only use fragments to surround the entire JSX in one element. You don't typically see them nested in the way you have them. You want to use typicaly HTML elements where you can.